src/functionalTest/java/uk/gov/hmcts/reform/pip/data/management/location/LocationTest.java

```diff
@@ -0,0 +1,137 @@
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.http.HttpStatus.OK;
+import static uk.gov.hmcts.reform.pip.data.management.utils.TestUtil.BEARER;
+
+@ExtendWith(SpringExtension.class)
+@ActiveProfiles(profiles = "functional")
+@AutoConfigureEmbeddedDatabase(type = AutoConfigureEmbeddedDatabase.DatabaseType.POSTGRES)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@SpringBootTest(classes = {Application.class, OAuthClient.class},
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class LocationTest extends FunctionalTestBase {
+
+    @Value("${system-admin-provenance-id}")
+    private String systemAdminProvenanceId;
+
+    private static final String LOCATION_LIST = "locationList";
+    private static final String TEST_LOCATION_NAME = "TestLocationOne";
+    private static final String TEST_LOCATION_ID = "2001";
+    private static final String TEST_JURISDICTION1 = "Magistrates";
+    private static final String TEST_JURISDICTION2 = "Family";
+    private static final String TEST_REGION = "North West";
+
+    private static final String BASE_LOCATIONS_URL = "/locations";
+    private static final String UPLOAD_LOCATIONS_URL = BASE_LOCATIONS_URL + "/upload";
+    private static final String GET_LOCATION_BY_ID_URL = BASE_LOCATIONS_URL + "/" + TEST_LOCATION_ID;
+    private static final String GET_LOCATION_BY_NAME_URL = BASE_LOCATIONS_URL + "/name/" + TEST_LOCATION_NAME
+        + "/language/ENGLISH";
+    private static final String GET_LOCATION_BY_REGION_AND_JURISDICTION = BASE_LOCATIONS_URL + "/filter";
+    private static final String DOWNLOAD_CSV_LOCATIONS_URL = BASE_LOCATIONS_URL + "/download/csv";
+
+    @Test
+    void locationControllerHappyPathTests() throws Exception {
+        Map<String, String> headerMap = new ConcurrentHashMap<>();
+        headerMap.put(AUTHORIZATION, BEARER + accessToken);
+
+        Map<String, String> deleteHeaderMap = new ConcurrentHashMap<>();
+        deleteHeaderMap.put(AUTHORIZATION, BEARER + accessToken);
+        deleteHeaderMap.put("x-provenance-user-id", systemAdminProvenanceId);
+
+        Map<String, String> headerMapUploadLocations = new ConcurrentHashMap<>();
+        headerMapUploadLocations.put(AUTHORIZATION, BEARER + accessToken);
+        headerMapUploadLocations.put("Content-Type", MediaType.MULTIPART_FORM_DATA_VALUE);
+
+        String filePath = this.getClass().getClassLoader().getResource("location/ValidLocations.csv").getPath();
+        File csvFile = new File(filePath);
+
+        final Response responseUploadLocations = doPostRequestMultiPart(
+            UPLOAD_LOCATIONS_URL,
+            headerMapUploadLocations, LOCATION_LIST, csvFile
+        );
+
+        assertThat(responseUploadLocations.getStatusCode()).isEqualTo(OK.value());
+        assertEquals(responseUploadLocations.as(Location[].class).length, 3,
```

Comment:
The assertion before this one is using assertJ. This assertion here is using JUnit assert. There are 2 different style of assertions in this test. Can you change this one to user assertJ too like below?

 assertThat(responseUploadLocations.as(Loaction[].class)).hasSize(3);

Reply:
Changed to assertJ.